### ISSUE TITLE
Can't authorize account on any version of Vivaldi

### ISSUE THREAD

**Issue by wilberfan** [flytaly/reddit-post-notifier#24]
The auth window opens and disappears in a split second.  Having this issue on both the latest Vivaldi Stable (v5.1.2567.49) and Vivaldi Snapshot (v5.2.2581.4).  Not sure if the problem is in the browser--or the extension.   Works properly on the latest Firefox Beta.

**Comment by flytaly** [comment]
Yeah, when extension calls [identity.launchWebAuthFlow](https://developer.chrome.com/docs/extensions/reference/identity/#type-WebAuthFlowDetails) for some reason it doesn't work properly with Reddit oauth page. 

It works in other chromium-based browsers, so I think it just Vivaldi's bug. 

As a workaround, you can import account data from a different browser. I can make instruction how to do it.

**Comment by flytaly** [comment]
The extension now can import/export configuration, so it's possible to add users from different browsers, save configuration and import it into Vivaldi.
`Include accounts data` should be checked.
![2022-03-23_14-26](https://user-images.githubusercontent.com/1577569/159689524-534935dc-74a2-4a2f-98ab-3013a8c774d6.png)
